cmd/composer-cli/blueprints/diff.go

```diff
@@ -25,7 +31,189 @@ func init() {
 }
 
 func diff(cmd *cobra.Command, args []string) (rcErr error) {
-	fmt.Printf("Ran the blueprints diff: %v command\n", args)
+	// args[0] == blueprint name, args[1] = FROM-COMMIT, args[2] = TO-COMMIT
+	// args[3] and later are passed to the diff executable
+	if args[1] == "WORKSPACE" {
+		return root.ExecutionError(cmd, "FROM-COMMIT cannot be WORKSPACE")
+	}
+
+	if root.JSONOutput {
```

Comment:
I wonder if we should attempt to diff the jsons in this case. It feels weird to me that a `diff` command in the json mode returns two undiffed blueprints. :thinking:

Reply:
Nope. The --json option should return the data from the server and not try to do anything with it other than pass it through to the user.